.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=true
```

Comment:
Do we need to enable package-lock here? Is that something we want to do throughout our repos?

Reply:
I enabled it to run `npm audit`.